app/services/service_status.service.js

```diff
@@ -37,70 +47,110 @@ class ServiceStatusService {
-      // * Its status, which is formatted numeric so that it's right justified on its row.
-      return [
-        { text: row[0] },
-        { text: row[1], format: 'numeric' }
-      ]
-    })
+  static async _getVirusScannerData () {
+    try {
+      const { stdout, stderr } = await exec('clamdscan --version')
+      return stderr ? `ERROR: ${stderr}` : stdout
+    } catch (error) {
+      return `ERROR: ${error.message}`
+    }
   }
 
-  static async _getImportData () {
-    return [
-      [
-        'Cell 1.1',
-        'Cell 1.2',
-        'Cell 1.3',
-        'Cell 1.4',
-        'Cell 1.5'
-      ],
-      [
-        'Cell 2.1',
-        'Cell 2.2',
-        'Cell 2.3',
-        'Cell 2.4',
-        'Cell 2.5'
-      ]
-    ]
+  static async _getRedisConnectivityData () {
+    try {
+      const { stdout, stderr } = await exec('redis-server --version')
+      return stderr ? `ERROR: ${stderr}` : stdout
+    } catch (error) {
+      return `ERROR: ${error.message}`
+    }
   }
 
-  static async _getVirusScannerData () {
-    return [
-      [
-        'Status',
-        'OK'
-      ]
-    ]
+  static async _getAddressFacadeData () {
+    const statusUrl = new URL('/address-service/hola', servicesConfig.addressFacade.url)
+    const result = await this._requestData(statusUrl)
+
+    return result.succeeded ? result.response.body : result.response
   }
 
-  static async _getCacheConnectivityData () {
-    return [
-      [
-        'Status',
-        'Connected'
-      ]
-    ]
+  static async _getChargingModuleData () {
+    const statusUrl = new URL('/status', servicesConfig.chargingModule.url)
+    const result = await this._requestData(statusUrl)
+
+    return result.succeeded ? result.response.headers['x-cma-docker-tag'] : result.response
   }
 
-  static async _getServiceVersionsData () {
-    return [
+  static async _requestData (url) {
+    // As of v12, the got dependency no longer supports CJS modules. This causes us a problem as we are locked into
+    // using these for the time being. Some workarounds are provided here: https://github.com/sindresorhus/got/issues/1789
+    // We have gone the route of using await import('got'). We cannot do this at the top level as Node doesn't support
+    // top level in CJS so we do it here instead.
+    const { got } = await import('got')
+    const result = {
+      succeeded: true,
+      response: null
+    }
+
+    try {
+      result.response = await got.get(url, {
+        retry: {
+          // We ensure that the only network errors Got retries are timeout errors
+          errorCodes: ['ETIMEDOUT'],
+          // We set statusCodes as an empty array to ensure that 4xx, 5xx etc. errors are not retried
+          statusCodes: []
+        }
+      })
+    } catch (error) {
+      const statusCode = error.response ? error.response.statusCode : 'N/A'
+      result.response = `ERROR: ${statusCode} - ${error.name} - ${error.message}`
```

Comment:
Is `ERROR: N/A - errorName - errorMessage` what we want to see if there's no status code or would we prefer to make it conditional so it's just `ERROR: errorName - errorMessage`? (Or do we not care since the page is just us? :grin:)

Reply:
I went with 'it's fine as it's just us'! I didn't want to complicate the code any more than I already had 😬